chain/validation/lotus_test.go

```diff
@@ -1,53 +1,129 @@
 package validation
 
 import (
+	"encoding/binary"
 	"math/big"
 	"testing"
 
+	"github.com/libp2p/go-libp2p-core/peer"
+	mh "github.com/multiformats/go-multihash"
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 
-	"github.com/filecoin-project/chain-validation/pkg/chain"
 	"github.com/filecoin-project/chain-validation/pkg/state"
 	"github.com/filecoin-project/chain-validation/pkg/suites"
+
+	"github.com/filecoin-project/lotus/chain/actors"
+	"github.com/filecoin-project/lotus/chain/address"
 )
 
-// A basic example validation test.
-// At present this code is verbose and demonstrates the opportunity for helper methods.
-func TestLotusExample(t *testing.T) {
+func TestStorageMinerValidation(t *testing.T) {
+	factory := NewFactories()
+	suites.CreateStorageMinerAndUpdatePeerIDTest(t, factory)
+
+}
+
+func TestValueTransfer(t *testing.T) {
 	factory := NewFactories()
-	drv := suites.NewStateDriver(t, factory.NewState())
+	suites.AccountValueTransferTest(t, factory)
```

Comment:
Corresponding test in chain-validation: https://github.com/filecoin-project/chain-validation/pull/12/files#diff-a132748ad00b6b23f7837c2c22bce156R14